wiki/addons/third-party/kubejs-irons-spells/page.kubedoc

```diff
@@ -101,9 +100,63 @@ StartupEvents.registry("irons_spellbooks:schools", event => {
 		.setFocus("kubejs:test_focus") // The focus needs to be an item tag. You can add this same tag to an item to make it the focus.
 		.setPowerAttribute(global.attributes.TEST_SPELL_POWER.get()) // This sets the power attribute. We use `.get()` on the attribute to obtain the registry object.
```

Comment:
Remove the `.get()` from attributes or anywhere.
We do not support those anymore because it is not safe.
Just the builderbase/registryobject is enough

Reply:
Don't remove the .get() from the ForgeEvent. At that point all registry is already granted to be safe.